### ISSUE TITLE
0.8.3 breaks Grunt task

### ISSUE THREAD

**Issue by dchw** [SitePen/remap-istanbul#116]
It seems like some change from 0.8.2 -> 0.8.3 has broken the Grunt task, I keep seeing the following error in our logs:

```
20-Dec-2016 19:13:09	Warning: Task "remapIstanbul" not found. Use --force to continue.
20-Dec-2016 19:13:09	
20-Dec-2016 19:13:09	Aborted due to warnings.
```

I can confirm that it works in 0.8.2, and I also can't see any changes I would need to make from your documentation [here](https://github.com/SitePen/remap-istanbul#grunt-task).  We are invoking remap from an npm script like this:
```javascript
"scripts": {
    "test": "istanbul cover --print none --report json --include-all-sources test/bootstrap.js && grunt remapIstanbul && istanbul report --include coverage-final.json --root coverage text text-summary html",
    ...
  },
```

Relevant portion of the gruntfile:
```javascript
module.exports = (grunt) => {
    grunt.initConfig({
    ...
        remapIstanbul: {
            build: {
                files: [{
                    src: 'coverage/coverage-final.json',
                    dest: 'coverage/coverage-final.json',
                    type: 'json'
                }]
            }
        },
    }
    ...
    grunt.loadNpmTasks('remap-istanbul');
}
```

**Comment by jdonaghue** [comment]
@dchw you are invoking it correctly. The issue is with a lingering `.default` in the grunt task left over from the `0.8.2` release. We will get this fixed soon. Sorry for the inconvenience!

**Comment by dchw** [comment]
@jdonaghue Cool, thanks. We (and our build system) look forward to a fix soon. For now we've locked all failing builds to use version 0.8.2 until this is fixed.

**Comment by jdonaghue** [comment]
@dchw This has been fixed and published to npm in version `0.8.4`. Some tests have been modified to ensure this doesn't happen again. Thanks!

**Comment by dchw** [comment]
Thanks!
